homeassistant/components/smartthings/const.py

```diff
@@ -9,6 +9,7 @@
 CONF_INSTALLED_APP_ID = 'installed_app_id'
 CONF_INSTANCE_ID = 'instance_id'
 CONF_LOCATION_ID = 'location_id'
+CONF_WEBHOOK_ID = 'webhook_id'
```

Comment:
This is defined in the homeassistant const file already. It's better to use it from there.

Reply:
Changed!  Good catch.